### DISCUSSION TITLE
About CSS

### DISCUSSION THREAD

**Discussion by GihanSanjeewa** [GihanSanjeewa/1mbee_main#2] [Q&A] [answered]
How To Use Animation in CSS ?

**Comment by UdaraEkanayake** [comment, marked as answer]
To use animation in CSS, you need to specify keyframes for the animation.The animation property in CSS can be used to animate many other CSS properties such as color, background-color, height, or width. Each animation needs to be defined with the @keyframes at-rule which is then called with the animation property.The animation-duration property defines how long the @keyframes timeline should be, which should be a time value.The overflow: hidden property can be used to move elements across the screen and create scrollbars on your web-thing. It's up to you to figure out when and how to use it.

**Reply by GihanSanjeewa** [reply]
Thank You Buddy
